### PR TITLE
Frontend tests

### PR DESCRIPTION
**Pull request by Charana123**

### CODE REVIEW

**Comment by matrixik** [comment]
Maybe stupid question but does anyone from stackhpc will be merging this PRs? Because now you are doing the same staff (like reformatting) again and I don't know JS at all...

**Comment by dougszumski** [comment]
@matrixik I don't want anything to be merged until I've tested it on a running Monasca deployment. Shouldn't be long now.

File: src/components/alarm_definitions.ts [hunk 1]
```diff
@@ -16,23 +16,42 @@
  */
 
 import appEvents from "app/core/app_events";
```
Comment (dougszumski) [inline src/components/alarm_definitions.ts]: This commit is doing a lot more than fixing a bug where the UI hangs. Really it should be a series of smaller commits.  At the very least please can we update the commit message?

File: src/components/alarm_definitions.html [hunk 2]
```diff
@@ -63,14 +63,14 @@ <h1 class="rt-h1">Alarm Definitions</h1>
 		  <i ng-class="icon" class="icon-gf icon-gf-settings" bs-tooltip="'Edit Alarm Definition'"></i>
 		</a>
 
-		<a ng-click="ctrl.enableAlarmDefinition(definition, true)" ng-if="definition.enabling">
-		  <i ng-class="icon" class="fa fa-spinner fa-spin" bs-tooltip="'Disabling Alarm Definition...'" ng-if="definition.actions_enabled"></i>
-		  <i ng-class="icon" class="fa fa-spinner fa-spin" bs-tooltip="'Enabling Alarm Definition...'" ng-if="!definition.actions_enabled"></i>
+		<a ng-if="definition.enabling">
+		  <i ng-class="icon" class="fa fa-spinner fa-spin" bs-tooltip="'Disabling Alarm Definition...'" ng-if="definition.actionsEnabled"></i>
```
Comment (dougszumski) [inline src/components/alarm_definitions.html]: Move refactoring into the refactoring commit.

File: src/components/alarm_definitions.ts [hunk 3]
```diff
@@ -68,27 +71,27 @@ export class AlarmDefinitionsPageCtrl {
   }
 
   // Enable Alarm Definitions
-  private setAlarmDefinitionActionsEnabled(id, actionsEnabled) {
+  private setAlarmDefinitionActionsToggleEnabled(id, actionsEnabled) {
```
Comment (dougszumski) [inline src/components/alarm_definitions.ts]: Move into refactoring commit

File: src/components/alarm_definitions.ts [hunk 4]
```diff
@@ -68,27 +71,27 @@ export class AlarmDefinitionsPageCtrl {
...
     var index = this.alarmDefinitions.findIndex(n => n.id === id);
     if (index !== -1) {
       this.alarmDefinitions[index].actionsEnabled = actionsEnabled;
     }
   }
 
-  private setAlarmDefinitionEnabling(id, enabling) {
+  private setAlarmDefinitionToggleEnabling(id, enabling) {
```
Comment (dougszumski) [inline src/components/alarm_definitions.ts]: Move into refactoring commit.

File: src/components/alarm_definitions.ts [hunk 5]
```diff
@@ -68,27 +71,27 @@ export class AlarmDefinitionsPageCtrl {
...
     var index = this.alarmDefinitions.findIndex(n => n.id === id);
     if (index !== -1) {
       this.alarmDefinitions[index].enabling = enabling;
     }
   }
 
-  private confirmEnableAlarmDefinition(id, actionsEnabled) {
-    this.setAlarmDefinitionEnabling(id, true);
+  private toggleEnableAlarmDefinition(id, actionsEnabled) {
+    this.setAlarmDefinitionToggleEnabling(id, true);
```
Comment (dougszumski) [inline src/components/alarm_definitions.ts]: Move into refactoring commit

Same for all other instances.

**Pull request merged**